spec/model/comment_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  # tests go here
+end
```

Comment:
Kindly add unit tests for comment model 👍 , you can test if the method inside comment model is working correctly